src/components/confirmVotes/confirmVotes.js

```diff
@@ -49,20 +56,24 @@ class ConfirmVotes extends React.Component {
       account,
       votes,
       passphrase: passphrase.value,
-      secondSecret: secondPassphrase.value,
+      secondPassphrase: secondPassphrase.value,
       goToNextStep: this.goToNextStep.bind(this),
     };
-
     return (
       <div className={styles.wrapper}>
         <article className={styles.content}>
-          <h2 className={styles.header}>{t('Final confirmation')}</h2>
+          <h2 className={styles.header}>{account.hwInfo && account.hwInfo.deviceId ?
+            t('Confirm on Ledger') : t('Final confirmation') }</h2>
```

Comment:
Should be more consistent with Send transaction:
```suggestion
            t('Confirm vote on Ledger Nano S') : t('Final confirmation') }</h2>
```